flow/Arena.h

```diff
@@ -631,6 +631,9 @@ struct Traceable<Standalone<T>> : std::conditional<Traceable<T>::value, std::tru
 };
 
 #define LiteralStringRef( str ) StringRef( (const uint8_t*)(str), sizeof((str))-1 )
+inline StringRef operator "" _ref(const char* str, size_t size) {
```

Comment:
@ajbeamon  Is this the decided name from your discussion with Markus?

Reply:
@satherton had proposed `sref` or `sr`, which I think sound good, but if `ref` ends up the chosen suffix I'm ok with that too.

Reply:
yes, it was not clear to me whether we reached a decision in that discussion. I changed it to `_sr` (`sr` and `sref` are both illegal UDLs) - I think this is the closest to a compromise and consistent with `std::string_view` and `sv`.